src/routes/Dashboard/RecentVehiclesCard.jsx

```diff
@@ -0,0 +1,49 @@
+import {useQuery} from "@apollo/client";
+import {DateTime} from "luxon";
+import SearchVehiclesCheckIns from "queries/SearchVehiclesCheckIns";
+import {useCallback, useEffect, useState} from "react";
+
+export default function RecentVehiclesCard(){
+    const [now, setNow] = useState(DateTime.now())
+    const updateNow = useCallback(() => setNow(DateTime.now()))
+
+    useEffect(() => {
+        const timeout = setTimeout(updateNow, 20000)
+
+        return () => clearTimeout(timeout)
+    }, [now])
+
+    const {loading, error, data} = useQuery(SearchVehiclesCheckIns)
+    if (loading) return <p>Loading...</p>;
+    if (error) return <p>Error :(</p>;
```

Comment:
Let's create an `/src/components/misc/ServerError.jsx` component and use it here, later on we're going to improve that component with a great UI to show better error messages.

Reply:
By now render "Ocurrió un error, recarga la página e intenta de nuevo"